jest_config/providersWrapper.tsx

```diff
@@ -10,8 +11,14 @@ import { theme } from '@theme';
   Custom wrapper to enable rendered tests to consume providers data
   Ref: https://testing-library.com/docs/react-testing-library/setup
 */
-export const ProvidersWrapper = ({ children }: { children: React.ReactNode }) => (
-  <Provider store={createStore().store}>
+export const ProvidersWrapper = ({
+  children,
+  initialState
+}: {
+  children: React.ReactNode;
+  initialState?: DeepPartial<AppState>;
+}) => (
+  <Provider store={createStore(initialState).store}>
     <ThemeProvider theme={theme}>{children}</ThemeProvider>
   </Provider>
 );
```

Comment:
Do we really need to pass the redux store to all the Components tests ?

Reply:
What do you mean? The Redux store must be there if you are testing a component that gets data from it

Reply:
Yes but it's not useful to load if the component don't use it :) I would prefer to wrap the component directly in the test file that needs it (like we do for `DataPriver` and `StoreProvider`) rather than loading this provider with every Component we test

Reply:
You still have to wrap it in `ProvidersWrapper` manually to use it. I didn't add the store, I just added the extra parameter

Reply:
`ProvidersWrapper` is used inside `render` in `test-utils`, it automatically wraps all the test renders with ProvidersWrapper :)

Reply:
Okay, but the we already did pass it to all components. I just added a prop

Reply:
Oh you're right ! I understand now why some of my test `console.warn` me on dumb components tests 😅 Nevermind then :)